src/server/auth/server/api_server.go

```diff
@@ -78,9 +78,8 @@ type apiServer struct {
 
 	// public addresses the fact that pachd in full mode initializes two auth
 	// servers: one that exposes a public API, possibly over TLS, and one that
-	// exposes a private API, for internal services. Only the public-facing auth
-	// service should export the SAML ACS and Metadata services, so if public
-	// is true and auth is active, this may export those SAML services
+	// exposes a private API, for internal services. Only one of these can launch
+	// the OIDC callback web server.
```

Comment:
👍 - good point highlighting that Callback Server can only be launched in one of the services